lib/generators/rodauth/templates/app/views/rodauth/tailwind/reset_password_request.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :title, rodauth.reset_password_request_page_title %>
+<div class="dark:bg-gray-900 min-h-full flex items-center justify-center py-4 px-4 sm:px-6 lg:px-8">
+  <div class="w-full max-w-sm">
+    <%= form_with url: rodauth.reset_password_request_path, method: :post, data: { turbo: false } do |form| %>
+      <h3 class="text-center mb-3 font-medium leading-tight text-3xl mt-0 mb-2 text-gray-700 dark:text-white">  Reset Password  </h3>
+      <div class="text-center text-sm text-gray-700 mb-8 dark:text-white">
+        <%== rodauth.reset_password_explanatory_text %>
+      </div>
+      <% if params[rodauth.login_param] && !rodauth.field_error(rodauth.login_param) %>
+        <%= form.hidden_field rodauth.login_param, value: params[rodauth.login_param] %>
+      <% else %>
+        <div class="py-4">
+          <%#= form.label "login", rodauth.login_label, class: "block text-sm text-gray-600" %>
+          <!-- The label is unecessary,  given the email placeholder below: -->
```

Comment:
Should we remove these comments?

Reply:
[fixed](https://github.com/janko/rodauth-rails/pull/114/commits/4bede9b975ca3fd7e812df14d56a709942516d94)